Foundation/Data.swift

```diff
@@ -790,7 +790,7 @@ public final class _DataStorage {
     }
 }
 
-internal class _NSSwiftData : NSData {
+public class _NSSwiftData : NSData {
```

Comment:
but if you `#if` guard it please provide a comment explaining why this is needed.